build.sbt

```diff
@@ -873,6 +873,8 @@ lazy val `logging-service` = project
       akkaHttp,
       "io.circe"              %%% "circe-core"   % circeVersion,
       "io.circe"              %%% "circe-parser" % circeVersion,
+      "junit"               % "junit"                 % junitVersion      % Test,
```

Comment:
something looks odd with formatting here

Reply:
there is `scalafmtSbt` command for that

Reply:
Turned out the SBT sources were never formatted. I created a separate PR for that #5880

Reply:
Reformatted in 0955b86313 - I hope there shall be no clashes with #5880 as the `build.sbt` was properly formatted before this PR.